api/v1alpha1/authenticationfilter_types.go

```diff
@@ -86,6 +99,11 @@ type JwtAuthenticationFilterProvider struct {
 	// HTTP/HTTPS endpoint.
 	RemoteJWKS RemoteJWKS `json:"remoteJWKS"`
 
+	// JWTClaimToHeaders is a list of values to extract the jwt claim value to the request header.
+	// For examples, following config:
+	// The claim must be of type; string, int, double, bool. Array type claims are not supported
```

Comment:
imo only string should be support, other fields will be expressed as string, right ?

Reply:
yeah,other fields will be expressed as string , but `Array` type claims are not supported in envoy.

Reply:
maybe just add that specific line, that "Array type claims are not supported"